CreeDictionary/API/models.py

```diff
@@ -133,7 +143,7 @@ class SearchResult:
 
     # Sequence of all preverb tags, in order
     # Optional: we might not have some preverbs in our database
-    preverbs: Tuple[Tuple[str, Optional[WordformID]], ...]
+    preverbs: Tuple[Preverb, ...]
```

Comment:
@eddieantonio could haved passed in Wordform. But I remembered the convention that dumb objects should be passed.

Reply:
Dumb objects _only when rendering a view_! Passing in a wordform here is okay!

Reply:
We do use SearchResult instances directly while rendering entries